task3/main_test.go

```diff
@@ -0,0 +1,55 @@
+package main
+
+import (
+	"testing"
+	_ "time"
+)
+
+func TestConvertToMilitaryTime(t *testing.T) {
+	type args struct {
+		time12format string
+	}
+	tests := []struct {
+		name    string
+		args    args
+		want    string
+		wantErr bool
+	}{
+		{
+			name:    "time1",
+			args:    args{time12format: "12:03:16PM"},
+			want:    "12:03:16",
+			wantErr: true,
+		},
+		{
+			name:    "time2",
+			args:    args{time12format: "12:45:13AM"},
+			want:    "00:45:13",
+			wantErr: true,
+		},
+		{
+			name:    "time3",
+			args:    args{time12format: ""},
```

Comment:
in my opinion need add extra check to func and if empty -> return error, like `couldn't convert empty line` 
also add new one test when you set not correct time